packages/dbos-compiler/generator.ts

```diff
@@ -0,0 +1,116 @@
+import path from 'node:path';
+import tsm from 'ts-morph';
+import { Liquid } from "liquidjs";
+import type { TransactionConfig, CompileResult } from './compiler.js';
+
+const __dirname = import.meta.dirname;
+const engine = new Liquid({
+  root: path.resolve(__dirname, '..', 'templates'),
+  extname: ".liquid"
+});
+
+async function render(file: string, ctx?: object): Promise<string> {
+  return await engine.renderFile(file, ctx) as string;
+}
+
+function mapType(type: tsm.Type) {
```

Comment:
If I passed the args in a json array, I wouldn't need to map types from javascript to sql